Sources/SymbolKit/SymbolGraph/Misc/SemanticVersion.swift

```diff
@@ -42,6 +42,8 @@ extension SymbolGraph {
             self.major = major
             self.minor = minor
             self.patch = patch
+            self.prerelease = prerelease
```

Comment:
@franklinsch Is this an intended behavior? Seems like a bug for me

Reply:
I agree, it seems like a bug. Is there a test we could add to exercise this code?

Reply:
I'll add a test file for it and update the PR